data/strings/strings.txt

```diff
@@ -28415,6 +28374,13 @@
     tr = https://wiki.openstreetmap.org/wiki/Tr:About
     uk = https://wiki.openstreetmap.org/wiki/Uk:Про_проект
 
+  [tts_info_guide_link]
+    comment = Link to OM TTS setup guide
+    tags = android
+    en = https://organicmaps.app/faq/text-to-speech-android-tts/
+    es = https://organicmaps.app/es/faq/text-to-speech-android-tts/
```

Comment:
@RedAuburn: Please fix the URL for FAQ in Spanish and add the URL for Italian...

```suggestion
    es = https://organicmaps.app/es/faq/instrucciones-de-voz-tts-en-android/
    it = https://organicmaps.app/it/faq/sintesi-vocale-tts-su-android/
```